frontend/src/components/layout/UserDropDown/UserDropDownContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import DropDown from 'components/Layout/UserDropDown/UserDropDown';
+import { User } from 'types/user';
+
+const UserDropDown: React.FC<User> = ({
+  userName,
+  userRole,
+  userUnit,
+  userDepartment,
+  userGroup,
+  avatarUrl,
+}) => {
+  const [isDropdownOpen, setDropdownState] = React.useState(false);
+  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
```

Comment:
Replace it with:
```suggestion
  const [isDropdownOpen, setDropdownState] = useState(false);
  const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
```
and
`import { useState } from 'react';`